analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.CSharp7.cs

```diff
@@ -259,7 +259,7 @@ public void Method4(object obj)
         default:
           break;
       }
-      var s = obj.ToString(); // Noncompliant FIXME (was compliant before)
+      var s = obj.ToString(); // Compliant
```

Comment:
Should be fixed by https://github.com/SonarSource/sonar-dotnet/pull/6047

Reply:
If I remember right, it was about like this: 
* It was introduced by you and then appeared in my PR after rebase. (That is when I made the comment)
* It was later fixed by you in #6047 which fixed it for me after rebase.